src/app/my-account/my-account/my-account.component.ts

```diff
@@ -61,14 +60,10 @@ export class MyAccountComponent implements OnInit, OnDestroy {
     private translationResourceService: TranslationResourceService,
     private otherResourceService: OtherResourceService,
     private translateService: TranslateService,
-    private readonly authService: AuthService,
     protected urlHelperService: UrlHelperService
   ) {}
 
   public async ngOnInit() {
-    if (!this.authService.isLoggedIn()) {
```

Comment:
To na pewno stąd wylatuje?

Reply:
Hej, tak sam to dodawałem przy okazji wylogowania na idle, widzę ze generuje pewne problemy, przynajmniej lokalnie, a raczej nie jest niezbędne bo wcześniej tego nie było.